karapace/backup/cli.py

```diff
@@ -34,6 +34,11 @@ def parse_args() -> argparse.Namespace:
     for p in (parser_get, parser_restore, parser_export_anonymized_avro_schemas):
         p.add_argument("--config", help="Configuration file path", required=True)
         p.add_argument("--topic", help="Kafka topic name to be used", required=False)
+        p.add_argument(
+            "--skip-topic-creation",
+            action="store_true",
+            help="Skip topic creation, restoring only the data into an already existing topic.",
```

Comment:
```suggestion
            help="Skip topic creation, restoring only the data into an already existing topic.",
            default=False,
```

issue: Without this I believe the annotation in `restore_backup` is not correct (`bool` vs `bool | None`).

Reply:
On the docs it says that when the action is `store_true`, the default is False. Anyway i've added it an explicit default=False so it's more clear.